### PR TITLE
Api stability common

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Comment by IgorMinar** [comment]
tests should be green now. fingers crossed.

File: modules/@angular/common/src/pipes/json_pipe.ts [hunk 1]
```diff
@@ -7,10 +7,10 @@ import {Json} from '../../src/facade/lang';
  *
  * ### Example
  * {@example core/pipes/ts/json_pipe/json_pipe_example.ts region='JsonPipe'}
+ *
+ * @experimental
```
Comment (mhevery) [inline modules/@angular/common/src/pipes/json_pipe.ts]: stable

File: modules/@angular/common/src/pipes/replace_pipe.ts [hunk 2]
```diff
@@ -35,6 +35,8 @@ import {InvalidPipeArgumentException} from './invalid_pipe_argument_exception';
  * --Note--: The 'pattern' parameter will be converted to a RegExp instance. Make sure to escape the
  * string properly if you are matching for regular expression special characters like parenthesis,
  * brackets etc.
+ *
+ * @experimental
```
Comment (mhevery) [inline modules/@angular/common/src/pipes/replace_pipe.ts]: deprecate

File: modules/@angular/common/src/pipes/slice_pipe.ts [hunk 3]
```diff
@@ -58,10 +58,11 @@ import {InvalidPipeArgumentException} from './invalid_pipe_argument_exception';
  * ## String Examples
  *
  * {@example core/pipes/ts/slice_pipe/slice_pipe_example.ts region='SlicePipe_string'}
+ *
+ * @stable
```
Comment (mhevery) [inline modules/@angular/common/src/pipes/slice_pipe.ts]: make it match `lowercase`

File: modules/@angular/common/src/pipes/uppercase_pipe.ts [hunk 4]
```diff
@@ -8,9 +8,10 @@ import {InvalidPipeArgumentException} from './invalid_pipe_argument_exception';
  * ### Example
  *
  * {@example core/pipes/ts/lowerupper_pipe/lowerupper_pipe_example.ts region='LowerUpperPipe'}
+ *
+ * @stable
```
Comment (mhevery) [inline modules/@angular/common/src/pipes/uppercase_pipe.ts]: match lowercase

File: modules/benchpress/src/validator.ts [hunk 5]
```diff
@@ -7,6 +7,8 @@ import {MeasureValues} from './measure_values';
  * A Validator calculates a valid sample out of the complete sample.
  * A valid sample is a sample that represents the population that should be observed
  * in the correct way.
+ *
+ * @experimental
```
Comment (mhevery) [inline modules/benchpress/src/validator.ts]: revert

**Pull request merged**